### PR TITLE
Create Is_prime.cpp

### PR DESCRIPTION
**Pull request by Tirtharaj-Talukdar**
Please accept my request I am a beginner

### CODE REVIEW

**Comment by Kumar-laxmi** [comment]
@Tirtharaj-Talukdar Please open an issue first addressing the work you are going to do

**Comment by Kumar-laxmi** [comment]
> @Tirtharaj-Talukdar Please open an issue first addressing the work you are going to do

"invalid" label will be removed after proper issue followed by PR is made

**Comment by Kumar-laxmi** [comment]
Kindly perform this algorithm using C, Python and Java

File: C++/Is_prime.cpp [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    //CHECK PRIME
+
+    // 1) O(N) complexity
+    /*
+    What we will do here is we will iterate from 2 till number -1
+    and check if the number is divisible by any of this. If it is then it is not
+    a prime number
+    */
+
+    for(int i=2;i<number;i++){
```
Comment (Kumar-laxmi) [inline C++/Is_prime.cpp:17]: Better implement this as one function

File: C++/Is_prime.cpp [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    // 2) O(sqrt(N)) complexity
+
+    /*
+    Here we will use the property that for every facto of a number 
+    less than its square root we get a factor which is greater than its squareroot.
+    So it is sufficient to check till its squareroot.
+    */
+
+    for(int i=2;i*i<=N;i++){
```
Comment (Kumar-laxmi) [inline C++/Is_prime.cpp:34]: And this logic as other function and just call the functions in main() method to make code look cleaner

**Comment by Tirtharaj-Talukdar** [comment]
> @Tirtharaj-Talukdar Kindly perform this algorithm using C, Python and Java to validate this PR

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
Please follow modular coding practice

**Comment by Tirtharaj-Talukdar** [comment]
Heyy I don't know any other language I can only write cpp

**Comment by Tirtharaj-Talukdar** [comment]
I can try to add Modular features but I can write only in cPP

File: C++/Is_prime.cpp (hunk 1 above)
Comment (Tirtharaj-Talukdar) [reply C++/Is_prime.cpp:17]: Okay sure trying to do

**Comment by Kumar-laxmi** [comment]
@Tirtharaj-Talukdar just give a try

File: C++/Is_prime.cpp (hunk 2 above)
Comment (Tirtharaj-Talukdar) [reply C++/Is_prime.cpp:34]: Okay Sure.

**Comment by Kumar-laxmi** [comment]
> @Tirtharaj-Talukdar just give a try

Other languages aren't hard enough

File: C++/Is_prime.cpp (hunk 1 above)
Comment (Kumar-laxmi) [reply C++/Is_prime.cpp:17]: Please re-open the PR and work! 😁

Comment (Tirtharaj-Talukdar) [reply C++/Is_prime.cpp:17]: I already made a new issue and made a PR there. It would be great if you could check that !

**Pull request closed without merging**